config/webpack.js

```diff
@@ -401,6 +401,10 @@ module.exports = (env) => {
         'process.env': JSON.stringify({ NODE_ENV: 'production' })
       }),
 
+      new CopyWebpackPlugin([
```

Comment:
This will still copy for beta (env != 'dev'). You need a special check just for `env.release`.